data/data_loader.py

```diff
@@ -39,8 +39,8 @@ def create_dataloaders(
     )
 
     train_loader = DataLoader(train_dataset, batch_size=config.batch_size, shuffle=True)
-    val_loader = DataLoader(val_dataset, batch_size=1, shuffle=False)
-    test_loader = DataLoader(test_dataset, batch_size=1, shuffle=False)
+    val_loader = DataLoader(val_dataset, batch_size=config.batch_size, shuffle=True)
```

Comment:
I have reintroduced shuffleing to the validation and test set.
We have now batching here aswell.